### PR TITLE
removed unpaid bookings from viewer.bookings query, hiding those book…

### PR DESCRIPTION
**Pull request by zach-at-the-skills**

### CODE REVIEW

File: server/routers/viewer.tsx [hunk 1]
```diff
@@ -295,6 +295,7 @@ const loggedInViewerRouter = createProtectedRouter()
             AND: [
               { NOT: { status: { equals: BookingStatus.CANCELLED } } },
               { NOT: { status: { equals: BookingStatus.REJECTED } } },
+              { NOT: { paid: { equals: false } } }, // hide unpaid bookings https://app.clickup.com/t/27pjd5c
```
Comment (mcasey8540) [inline server/routers/viewer.tsx]: @zach-at-the-skills what about `{ NOT: { confirmed: { equals: false } } } `

**Review by mcasey8540 — CHANGES_REQUESTED** [review]

File: server/routers/viewer.tsx (hunk 1 above)
Comment (zach-at-the-skills) [reply server/routers/viewer.tsx]: Seems equivalent, but maybe that will be better for the future?

File: server/routers/viewer.tsx [hunk 2]
```diff
@@ -295,6 +295,7 @@ const loggedInViewerRouter = createProtectedRouter()
             AND: [
               { NOT: { status: { equals: BookingStatus.CANCELLED } } },
               { NOT: { status: { equals: BookingStatus.REJECTED } } },
+              { NOT: { confirmed: { equals: false } } }, // hide unconfirmed bookings https://app.clickup.com/t/27pjd5c
```
Comment (mcasey8540) [inline server/routers/viewer.tsx]: Are paid:true bookings always marked as confirmed?

Comment (mcasey8540) [reply server/routers/viewer.tsx]: @zach-at-the-skills add back `{ NOT: { paid: { equals: false } } }` in addition to `{ NOT: { confirmed: { equals: false } } },` to cover all use-cases

**Review by mcasey8540 — CHANGES_REQUESTED** [review]

**Review by mcasey8540 — APPROVED** [review]

**Pull request merged**